dali/python/nvidia/dali/plugin/tf.py

```diff
@@ -30,7 +30,23 @@
 
 _dali_tf = _dali_tf_module.dali
 
+def DALIIteratorWrapper(pipeline, **kwargs):
```

Comment:
I think it would be better to drop the positional "pipeline" argument and check kwargs for presence of the old "serialized_pipeline" or the new "pipeline" argument. Based on that, we can select old or the proposed behaviour without breaking existing code.

Reply:
Done